tests/integration/test_rpm.py

```diff
@@ -108,6 +111,56 @@
     )
 
 
+@pytest.mark.skipif(os.getenv("CACHI2_TEST_LOCAL_DNF_SERVER") != "true", reason="CACHI2_TEST_LOCAL_DNF_SERVER != true")
+def test_cert_auth(cachi2_image: utils.ContainerImage,
+                   tmp_path: Path,
+                   test_data_dir: Path,
+                   request: pytest.FixtureRequest) -> None:
+
+    client_cert = "/certificates/testuser.crt"
+    client_key = "/certificates/testuser.key"
+    ca_bundle = "/certificates/myCA.crt"
+
+    print(client_cert)
```

Comment:
@eskultety this will need an override; it is necessary for the integration test and these are not sensitive certs, but ones generated specifically for this test.